gossip/src/cluster_info.rs

```diff
@@ -2422,6 +2421,20 @@ impl ClusterInfo {
             }
             Ok(())
         };
+        let mut pings = Vec::new();
```

Comment:
we should probably consider wrapping the ping broadcast logic and cache in a service. seems brittle to maintain it manually like this